buildout.cfg

```diff
@@ -52,3 +52,4 @@ zope.interface = 4.1.2
 # zc.recipe.testrunner==2.0.0
 zope.testrunner = 4.4.6
 
+docker-pycreds = 0.3.0
```

Comment:
Was this added intentionally?

Reply:
Yes, it showed up a missing nailed dependency when I ran buildout.
I don't care, leaving it as it was, since I would like to get rid of buildout anyway ....